src/unitxt/catalog/cards/llm_as_judge/model_response_assessment/mt_bench.json

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "task_card",
+    "loader": null,
+    "preprocess_steps": [],
+    "task": "tasks.llm_as_judge.rag.model_response_assessment",
+    "templates": [
+        "templates.llm_as_judge.model_response_assessment.mt_bench"
```

Comment:
templates.rag.model_response_assessment.llm_as_judge_using_mt_bench_template

Reply:
but its not rag. There is no context

Reply:
also no "answer"

Reply:
You are right.  This template fits any task that has a question fields.  So it could be called

templates.qa.model_response_assessment.llm_as_judge_using_mt_bench_template

we probably should have alias for both - so people people will be able to find it.

templates.qa.model_response_assessment.llm_as_judge_using_mt_bench_template
templates.rag.model_response_assessment.llm_as_judge_using_mt_bench_template

Thinking more maybe rag task is a special kind of qa task so we should change the hierarchy (not for this PR).

 task.qa.rag
 task.qa.open
 task.qa.multiple-choice